app/src/index.js

```
@@ -35,7 +35,21 @@ import {
   wReady,
   wAdd_EvtApp,
 } from "./Db.js";
-import { wHandlePaypalCreateOrder, wHandlePaypalCaptureOrder } from "./Payment.js";
+import {
+  wHandlePaypalCreateOrder,
+  wHandlePaypalCaptureOrder,
+} from "./Payment.js";
```

ESLint

[error] Replace `⏎··wHandlePaypalCreateOrder,⏎··wHandlePaypalCaptureOrder,⏎` with `·wHandlePaypalCreateOrder,·wHandlePaypalCaptureOrder·`

(prettier/prettier)